experimental/language-generation/javascript_nodejs/13.core-bot/package.json

```diff
@@ -17,11 +17,11 @@
     },
     "dependencies": {
         "@microsoft/recognizers-text-data-types-timex-expression": "^1.1.4",
-        "botbuilder": "~4.7.0",
-        "botbuilder-ai": "~4.7.0",
-        "botbuilder-dialogs": "~4.7.0",
-        "botbuilder-testing": "~4.7.0",
-        "botbuilder-lg": "4.7.0-preview",
+        "botbuilder": "~4.8.0",
+        "botbuilder-ai": "~4.8.0",
+        "botbuilder-dialogs": "~4.8.0",
+        "botbuilder-testing": "~4.8.0",
+        "botbuilder-lg": "4.8.0-preview",
```

Comment:
>preview [](start = 32, length = 7)

I thought this was rc1?

Reply:
The package is labeled preview in JS. Not sure why. Rather than opening up a drill to re-publish, I think having RC in docs is ok. Let me know if you think otherwise.